runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -613,6 +618,23 @@ fn contract_compile_base_per_byte_v2(ctx: &mut EstimatorContext) -> (GasCost, Ga
     ctx.cached.compile_cost_base_per_byte_v2 = Some(costs.clone());
     costs
 }
+fn pure_deploy_bytes(ctx: &mut EstimatorContext) -> GasCost {
+    let small_code = generate_data_only_contract(0);
+    let large_code = generate_data_only_contract(bytesize::mb(4u64) as usize);
+    let small_code_len = small_code.len();
+    let large_code_len = large_code.len();
+    let cost_empty = deploy_contract_cost(ctx, small_code, Some(b"main"));
+    let cost_4mb = deploy_contract_cost(ctx, large_code, Some(b"main"));
+
+    if cost_4mb < cost_empty {
+        eprintln!("High variance in pure_deploy_bytes: Deployment cost of small contract is higher than large contract.");
+        let mut cost = GasCost::zero(ctx.config.metric);
+        cost.set_uncertain(true);
```

Comment:
Do we need a `eprintln` here? Human attention is a very scarce resource, so any ad hoc printouts have a very high cost to them. 

Maybe we need `cost.set_uncertain("high variance in ... ");` instead, and display this when printing the cost? 

If ^ is a good idea, it's probably do the refacto in a follow-up though.

Reply:
I think we can just remove the output. A literal zero cost + uncertain flag in the output seems good enough to avoid confusing the output with an actual measurement. Latest commit has it removed.

We can consider making the uncertain flag more sophisticated. But it forces us to come up with some rule how we treat it in addition/subtraction or how we reduce many flags into one. That's the main reason I like the simple boolean, it keeps the `GasCost` closer to a simple primitive.